src/components/select/select-list/select-list.component.js

```diff
@@ -66,6 +70,16 @@ const SelectList = React.forwardRef(
     const listActionButtonRef = useRef();
     const { blockScroll, allowScroll } = useScrollBlock();
 
+    const virtualizer = useVirtualizer({
+      count: React.Children.count(children),
+      getScrollElement: () => listContainerRef.current,
```

Comment:
Wouldn't you be able to avoid overriding the `virtualizer.scrollToIndex` method if you would pass `tableRef` (`StyledSelectListTableBody`) as a scroll element when in `multiColumn` mode?